2021/day11/day11.java

```diff
@@ -0,0 +1,148 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class day11 {
+    /*
+     * Creates a grid surrounded by a border of nines, e.g.:
+     * 999999
+     * 1234 912349
+     * 5678 --> 956789
+     * 9999 999999
+     * 999999
+     */
+    public static ArrayList<ArrayList<Integer>> readInput(String filename) {
+        ArrayList<ArrayList<Integer>> input = new ArrayList<>();
+        ArrayList<Integer> emptyLine = new ArrayList<>();
+        Scanner sc = null;
+        try {
+            sc = new Scanner(new File(filename));
+            sc.useDelimiter("");
+            input.add(emptyLine);
+            while (sc.hasNextLine()) {
+                ArrayList<Integer> line = new ArrayList<>();
+                line.add(9);
+                while (sc.hasNextInt()) {
+                    line.add(sc.nextInt());
+                }
+                line.add(9);
+                input.add(line);
+                if (sc.hasNextLine())
+                    sc.nextLine();
+            }
+
+            input.add(emptyLine);
+            for (int i = 0; i < input.get(1).size(); i++) {
+                input.get(0).add(9);
+                input.get(input.size() - 1).add(9);
+            }
+
+        } catch (Exception e) {
+            System.out.println("Something went wrong " + e);
+        } finally {
+            sc.close();
+        }
+        return input;
+    }
+
+    public static void initialIncrease(ArrayList<ArrayList<Integer>> input) {
```

Comment:
try to use verbs in the method names. I have no idea how to name this one though lol. Also `input` -> `octoGrid`